agent/lib/kontena/agent.rb

```diff
@@ -6,62 +6,88 @@ class Agent
 
     VERSION = File.read('./VERSION').strip
 
+    # Called from other actors
+    def self.shutdown
+      @@instance.write_signal('shutdown')
+    end
+
     def initialize(opts)
+      @@instance = self
```

Comment:
Means that it's now `configure(opts)`, because it doesn't look like a `Singleton` can be initialized with parameters?